### PR TITLE
Adds 6 new psionic powers

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Locked away behind ponderous deepmaints mobs are three new trash based psionic powers
One psionic power will exploid any trash pile infrount of the player
One psionic power will instantly cube a trash pile infrount of a player 
and one psionic power will spawn a trash pile infrount of a player
Locked away behind flesh behemoth deepmaints mobs are three new psionic powers
One psionic power set your temp to default mob and clear frost
One psionic power remove all chemicals in your bloodstream, on your skin, and what you have eaten
and one will spawn some psionic needle and thread to help stop bleeding, if you have a few psionic perks it even heals your a little.

### CODE REVIEW

File: code/modules/psionics/psion_powers/trash_powers.dm [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	if(PT && PT.pay_power_cost(psi_point_cost) && PT.check_possibility())
+		var/turf/T = get_step(user, user.dir)
+		if(T)
+			to_chat(user, "Your mind works to geather random junk fallin deepmaints and bring them to your current location.")
+			new /obj/random/scrap/moderate_weighted(T.contents)
+
+/mob/living/carbon/human/psionic_tumor/proc/trash_pile_compress()
+	set category = "Psionic powers"
+	set name = "Reorganize (2)"
+	set desc = "Exspend a small amount of essene to compress a scrap pile into a condenced cube for easyer transportation."
```
Comment (benj8560) [inline code/modules/psionics/psion_powers/trash_powers.dm:22]:
```suggestion
	set desc = "Expend a small amount of essence to compress a scrap pile into a condensed cube for easier transportation."
```

File: code/modules/psionics/psion_powers/trash_powers.dm [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	var/obj/item/organ/internal/psionic_tumor/PT = user.first_organ_by_process(BP_PSION)
+
+	if(user.stats.getPerk(PERK_PSI_ATTUNEMENT))
+		to_chat(user, "Your psionic attunement allows save an essence.")
+		psi_point_cost = 3
+
+	if(PT && PT.pay_power_cost(psi_point_cost) && PT.check_possibility())
+		var/turf/T = get_step(user, user.dir)
+		if(T)
+			to_chat(user, "Your mind works to geather random junk fallin deepmaints and bring them to your current location.")
```
Comment (benj8560) [inline code/modules/psionics/psion_powers/trash_powers.dm:16]:
```suggestion
			to_chat(user, "Your mind works to gather random junk fallin in deepmaints to bring them to your current location.")
```

File: code/modules/psionics/psion_powers/trash_powers.dm [hunk 3]
```diff
@@ -0,0 +1,67 @@
+/mob/living/carbon/human/psionic_tumor/proc/summan_trash_pile()
+	set category = "Psionic powers"
+	set name = "Scrap Togather (4)"
+	set desc = "Exspend some essence to collect trash from deepmaints and pile it infrount of you."
```
Comment (benj8560) [inline code/modules/psionics/psion_powers/trash_powers.dm:4]:
```suggestion
	set desc = "Expend some essence to collect trash from deepmaints and pile it infrount of you."
```

File: code/modules/psionics/psion_powers/trash_powers.dm [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+	if(PT && PT.pay_power_cost(psi_point_cost) && PT.check_possibility())
+		var/turf/T = get_step(user, user.dir)
+		if(T)
+			var/refund = TRUE
+			for(var/obj/structure/scrap/jnk in T.contents)
+				jnk.ex_act(1)
+				refund = FALSE
+				break
+			if(refund)
+				to_chat(user, "Do to being unable to rapidly decompress a scrap pile you dont waste your essene.")
```
Comment (benj8560) [inline code/modules/psionics/psion_powers/trash_powers.dm:65]:
```suggestion
				to_chat(user, "Due to being unable to rapidly decompress a scrap pile you don't waste your essence.")
```

File: code/modules/psionics/psion_powers/trash_powers.dm [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+	if(PT && PT.pay_power_cost(psi_point_cost) && PT.check_possibility())
+		var/turf/T = get_step(user, user.dir)
+		if(T)
+			var/refund = TRUE
+			for(var/obj/structure/scrap/jnk in T.contents)
+				jnk.make_cube()
+				refund = FALSE
+				break
+			if(refund)
+				to_chat(user, "Do to being unable to compress down a scrap pile you dont waste your essene.")
```
Comment (benj8560) [inline code/modules/psionics/psion_powers/trash_powers.dm:40]:
```suggestion
				to_chat(user, "Due to being unable to compress down a scrap pile you dont waste your essence.")
```

**Review by benj8560 — COMMENTED** [review]
Believe I got everything

File: code/modules/psionics/psion_powers/trash_powers.dm [hunk 6]
```diff
@@ -0,0 +1,67 @@
+/mob/living/carbon/human/psionic_tumor/proc/summan_trash_pile()
+	set category = "Psionic powers"
+	set name = "Scrap Togather (4)"
+	set desc = "Expend some essence to collect trash from deepmaints and pile it in front of you."
```
Comment (benj8560) [inline code/modules/psionics/psion_powers/trash_powers.dm:4]:
```suggestion
	set desc = "Expend some essence to collect trash from deepmaints and pile it in front of you."
```

File: code/modules/psionics/psion_powers/trash_powers.dm [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+	if(PT && PT.pay_power_cost(psi_point_cost) && PT.check_possibility())
+		var/turf/T = get_step(user, user.dir)
+		if(T)
+			to_chat(user, "Your mind works to gather random junk fallin in deepmaints to bring them to your current location.")
+			new /obj/random/scrap/moderate_weighted(T.contents)
+
+/mob/living/carbon/human/psionic_tumor/proc/trash_pile_compress()
+	set category = "Psionic powers"
+	set name = "Reorganize (2)"
+	set desc = "Expend a small amount of essence to compress a scrap pile into a condensed cube for easier transportation."
```
Comment (benj8560) [inline code/modules/psionics/psion_powers/trash_powers.dm:22]:
```suggestion
	set desc = "Expend a small amount of essence to compress a scrap pile into a condensed cube for easier transportation."
```

File: code/modules/psionics/psionic_items/psi_catalysts.dm [hunk 8]
```diff
@@ -168,6 +168,49 @@
...
+	desc = "Psionic catalysts, other worldly items not quite understood, but valuable for the powers they may grant a psion. To everyone else, they have research value in a deconstructor or may be \
+	recycled for the somewhat rare materials that make them. Holding it feels quite strange. Even to an unattuned mind, one can hear the faintly glowing object whispering, the eager voices say: \
+	One persons trash is anothers treaser, lets dig for some!"
+	stored_power = /mob/living/carbon/human/psionic_tumor/proc/summan_trash_pile
+
+/obj/item/device/psionic_catalyst/needle_n_thread
+	name = "psionic catalyst: Psionic Suture"
+	desc = "Psionic catalysts, other worldly items not quite understood, but valuable for the powers they may grant a psion. To everyone else, they have research value in a deconstructor or may be \
+	recycled for the somewhat rare materials that make them. Holding it feels quite strange. Even to an unattuned mind, one can hear the faintly glowing object whispering, the eager voices say: \
+	A healthy body dosn't bleed..."
```
Comment (benj8560) [inline code/modules/psionics/psionic_items/psi_catalysts.dm:196]:
```suggestion
	A healthy body doesn't bleed..."
```

File: code/modules/psionics/psionic_items/psi_catalysts.dm [hunk 9]
```diff
@@ -168,6 +168,49 @@
...
+	stored_power = /mob/living/carbon/human/psionic_tumor/proc/needle_n_thread
+
+/obj/item/device/psionic_catalyst/purefie
+	name = "psionic catalyst: Psionic Purefie"
+	desc = "Psionic catalysts, other worldly items not quite understood, but valuable for the powers they may grant a psion. To everyone else, they have research value in a deconstructor or may be \
+	recycled for the somewhat rare materials that make them. Holding it feels quite strange. Even to an unattuned mind, one can hear the faintly glowing object whispering, the eager voices say: \
+	A healthy body dosn't have junk in it's systems..."
```
Comment (benj8560) [inline code/modules/psionics/psionic_items/psi_catalysts.dm:203]:
```suggestion
	A healthy body doesn't have junk in it's systems..."
```

File: code/modules/psionics/psionic_items/psi_catalysts.dm [hunk 10]
```diff
@@ -168,6 +168,49 @@
...
+	stored_power = /mob/living/carbon/human/psionic_tumor/proc/purefie
+
+/obj/item/device/psionic_catalyst/temp_regulate
+	name = "psionic catalyst: Psionic Temperature Regulate"
+	desc = "Psionic catalysts, other worldly items not quite understood, but valuable for the powers they may grant a psion. To everyone else, they have research value in a deconstructor or may be \
+	recycled for the somewhat rare materials that make them. Holding it feels quite strange. Even to an unattuned mind, one can hear the faintly glowing object whispering, the eager voices say: \
+	A healthy body dosn't sweat or shiver..."
```
Comment (benj8560) [inline code/modules/psionics/psionic_items/psi_catalysts.dm:210]:
```suggestion
	A healthy body doesn't sweat or shiver..."
```

**Comment by cdb-is-not-good** [comment]
for deepmaint powers these are fine.f

**Pull request merged**